### ISSUE TITLE
Container time and log file timestamps missmatch

### ISSUE THREAD

**Issue by wechsler42** [Forceu/barcodebuddy-docker#13]
Hi there,
without setting the TZ environment variable, I get:
root@0bd1db695c6f:/# date
Sat Jan 30 12:32:33 UTC 

If I set the environment variable 
TZ=Europe/Berlin (UTC+1 at the moment)
I get:
root@752f4f15e526:/# date
Sat Jan 30 13:33:49 CET 2021

But in the Web UI, the log of processed barcodes shows in both cases the UTC time, see for example:
2021-01-30 12:34:36: Unknown product already scanned. Increasing quantity. Barcode: XXXXXXXXXX

How to get the barcode buddy log to respect the timezone of the container?
Thanks in advance for a hint.

**Comment by Forceu** [comment]
I just tried it and unfortunately I was unable to reproduce the problem. Please note that the date of previous entries is not changed in case you imported the database.

**Comment by wechsler42** [comment]
Thanks Forceu for the hint. After backing up my data directory I ditched the whole config directory on the host and started again with the environment variable  "TZ=Europe/Berlin " set. And yes, it works now :-) Then I overwrote "barcodebuddy.db" with the one I backed up, restarted the container and it works flawlessly. The timezone is respected in the log file entries.
The difference to the situation before is, that on my first trial, I started without the setting the timezone and introduced it later on when the database was already filled with some entries. This seems to prevent the log file entries to respect the correct timezone set in the container.
